staging/src/k8s.io/api/admissionregistration/v1/generated.pb.go

```diff
@@ -342,13 +342,13 @@ func init() {
 }
 
 var fileDescriptor_aaac5994f79683e8 = []byte{
-	// 1104 bytes of a gzipped FileDescriptorProto
+	// 1102 bytes of a gzipped FileDescriptorProto
```

Comment:
whoa... didn't expect these generation changes... any idea what happened here?

Reply:
to be honest no 😄

Reply:
hmm... did we accidentally bump proto generator libraries somehow in one of the build images?

Reply:
I will go over the PRs, but I don't remember bumping this

for reference:
- https://github.com/kubernetes/release/pull/1857
- https://github.com/kubernetes/repo-infra/pull/223

Reply:
> hmm... did we accidentally bump proto generator libraries somehow in one of the build images?

Potentially a result of this PR to enable more arches for kube-cross: https://github.com/kubernetes/release/pull/1853

cc: @dims

Reply:
@liggitt @justaugustus yep, that's the one. how bad is it?

Reply:
ok, that explains the diff. would like an ack from node API reviewers that the change in the generated protobuf tag to drop json field name doesn't affect their use